node-runner-cli/commands/ledgercommand.py

```diff
@@ -0,0 +1,84 @@
+from argparse import ArgumentParser
+from argparse import RawTextHelpFormatter
+
+import botocore
+
+from commands.subcommand import get_decorator, argument
+
+import boto3
+
+ledgercli = ArgumentParser(
+    description='Subcommand to help to sync up the ledger from a S3 bucket',
+    usage="radixnode ledger ",
+    formatter_class=RawTextHelpFormatter)
+ledger_parser = ledgercli.add_subparsers(dest="ledgercommand")
+
+
+def ledgercommand(ledgercommand_args=[], parent=ledger_parser):
+    return get_decorator(ledgercommand_args, parent)
+
+
+@ledgercommand([
+    argument("-d", "--dest",
+             help="Destination path where the backup of the ledger will be downloaded ",
+             action="store", 
+             default="./"),
```

Comment:
Might be better we use python way of referring local directory as './' is very specific to unix